.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '10.x'
+    - name: Get package manager's global cache path
+      id: global-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - name: Cache package manager's global cache
+      id: cache-global-package-manager-cache
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.global-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-yarn-
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Install Dependencies
+      run: yarn install --frozen-lockfile
+      if: |
+        steps.cache-global-package-manager-cache.outputs.cache-hit != 'true' ||
+        steps.cache-node-modules.outputs.cache-hit != 'true'
+    - name: Lint
+      run: yarn lint
+
+  test:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    needs: lint
```

Comment:
To confirm, did you want to:

(a) run `lint` and `test` in series (current approach, where `test` occurs after `lint` succeeds), or
(b) run in parallel (`test` starts regardless of whether `lint` succeeds)?